Add 2026 CFP (#4)
cucapra/latte24: 2026/web/content/_index.md

+++
+++

LATTE is a venue for discussion, debate, and brainstorming at the intersection of hardware acceleration and programming languages research. The core mission is to bring ideas we love from software programming languages and tools to the world of hardware design.

> LATTE is a venue for discussion, debate, and brainstorming at the intersection of hardware acceleration and programming languages research. The core mission is to bring ideas we love from software programming languages and tools to the world of hardware design. For more details on the workshop's focus, see [the call for papers][cfp].

Reviewer: Am I misreading it, or are there two copies here?

Author: Oh good catch, that is remnants from last year

## The 2026 Workshop

> LATTE is a hybrid workshop; please consider attending either in person (in Rotterdam, co-located with [ASPLOS][]) or virtually (on Zoom). LATTE will be held on March 30.

Reviewer: Location needs updating? "ASPLOS 2026 will be in Pittsburgh, USA, from March 22nd to 26th, 2026"

- Early, in-progress research snapshots
- Experience reports on building or deploying accelerators and the tools involved
- Essays advocating for or against a general approach
- Retrospectives on past efforts on tools, languages, and techniques for accelerator design
- Calls for solutions to open challenges in the area (questions without answers)
- Demonstrations of real systems (to be shown off in a live demo at the workshop)
- Overviews of open-source projects, even when the novelty with respect to proprietary alternatives is limited

### Position Papers

The primary goal of the workshop is to enable discussion. It will accept **2-page position papers**.
The workshop will allocate short time slots to the papers, each paired with a discussion following [SNAPL][]'s discussion format:
“table discussion” where small breakout groups will discuss the paper, followed by plenary Q&A.

In order to make sure that papers are aligned with the topics of the workshop, they should explicitly address one (or more!) of these themes

- Hardware design tools MUST change
- Verification REMAINS the key challenge

> - The tools we build NEED to interoperate with each other to be successful

Reviewer 2: ```suggestion
- The tools we build NEED to interoperate with each other to be successful
- A *clearly and concisely articulated* theme of your own choosing. Apply the following tests when creating a theme: (1) does it apply more broadly than just your paper, and (2) can it be articulated with a short sentence.
```

> - The tools we build NEED to interoperate with each other to be successful

Reviewer: What was the theme/idea of the capitalized words again?

Author: Emphasis I suppose, but we can also just remove it

Reviewer: Cool, was just a bit confused looking at it from a cold eye again

For example, papers presenting a new tool or language are still encouraged, but rather than just presenting your new tool, try to make it fit with the themes by arguing that "Hardware description languages must change to address X,  that is why Y does Z".

> In addition, we are looking for papers that propse new "themes" like these. The idea is to highlight key questions that we as a community should try to address. These papers will be given more time for deeper discussion during the workshop, and as such will be held to a higher standard than normal papers.

Reviewer 2: I'm not sure about the "higher standard" part. I have proposed a change above (which we can move here instead) that talks about some tests being applied. Perhaps we can say that we will apply these tests to such "theme proposing" papers? This gives the authors a clear target to aim for as well instead of making up their own higher standard!

Author: Yep, good point, I wasn't very happy with that phrasing either

**Important guidelines.**
It's standard for papers to start with a general motivation: Moore's Law is doomed; specialized hardware is ascendant; Verilog is hard to use; etc.
*Please skip this part* in your LATTE position paper (and in eventual talks at the workshop).
The LATTE audience will already believe these things, so save the space & time and instead focus on your own unique ideas.
As much as possible, dispose with the framing and motivation so you can focus on the technical content.

Remember that the goal at LATTE is to stimulate discussion, not to disseminate fully polished results.
So don't be afraid to write up half-baked ideas and in-progress work: it's OK if your submission has zero bar charts, for example.

For examples of past position papers, consider looking at the programs for [LATTE ’25][latte-25] [LATTE ’24][latte-24], [’23][latte-23], [’22][latte-22], and [’21][latte-21].

**NOTE**: Using an LLM to fully generate your paper is ground for desk rejection. Using an LLM for small edits is fine, but keep in mind that we want to hear your ideas a phrased by you, not by the LLM.

> We are also not interested in LLM-focused research. LATTE is a venue for discussing languages and tools by and for humans.

Reviewer 2: ```suggestion
We are also not interested in LLM-focused research. LATTE is a venue for discussing languages and tools by humans and for humans.
```

> **NOTE**: Using an LLM to fully generate your paper is ground for desk rejection. Using an LLM for small edits is fine, but keep in mind that we want to hear your ideas a phrased by you, not by the LLM.

Reviewer: ```suggestion
**NOTE**: Using an LLM to fully generate your paper is ground for desk rejection. Using an LLM for edits is fine, but keep in mind that we want to hear your ideas expressed by you, not by the LLM.
```

Author: How about changing "small" to "stylistic" or something instead of removing it entirely?

Reviewer: SGTM

Also I just realised, `is ground for desk rejection` -> `is grounds for desk rejection`

Author: And that's why it's good to have a native speaker check things :D

## Hybrid Format

> LATTE 2026 will run in a hybrid format, i.e., it will be possible to join either in person (in Rotterdam) or online (on Zoom).

Reviewer: ```suggestion
LATTE 2026 will run in a hybrid format, i.e., it will be possible to join either in person or online (on Zoom).
```

[latte-24]: https://capra.cs.cornell.edu/latte24/

[cfp]: @/cfp.md
[asplos]: https://www.asplos-conference.org/asplos2026/
[matrix]: https://matrix.to/#/#latte-chat:fossi-foundation.org

## Attending LATTE

LATTE is a hybrid workshop; please consider attending either in person (in Rotterdam, co-located with [ASPLOS][]) or virtually (on Zoom). LATTE will be held on March 30.

* To attend in person, [register to attend ASPLOS workshops][asplos-reg].
* To attend virtually, no conference registration is required. Please [sign up using this Zoom form][zoom] to receive a link to join.

[asplos-reg]: https://www.asplos-conference.org/asplos2026/registration/
[zoom]: https://cornell.zoom.us/meeting/register/UfYN6ksoTLSbmm7gO1ApOA

## LATTE Matrix Chat

We have set up a Matrix chat to keep the LATTE community in touch outside the
main latte event. [Join it here.][matrix] Any vaguely LATTE-related discussion
is encouraged! Showcase cool projects you built, discuss new ideas or problems
you've run into, or just hang out with the community.
We'll also use this for real-time communication during the workshop.

## Organizers

LATTE's organizing team this year is:

> * [Rachit Nigam](https://rachit.pl), MIT & Jane Street

Reviewer 2: Just MIT now!

* [Adrian Sampson](https://www.cs.cornell.edu/~asampson/), Cornell

> * [Frans Skarman](https://liu.se/en/employee/frask53), Munich University of Applied Sciences

Reviewer: Frans - do you have a new homepage outside of Linkoping now?
